_posts/2022-09-03-aphp-user-story.adoc

```diff
@@ -0,0 +1,74 @@
+---
+layout: post
+title: 'Quarkus adoption by APHP (Assistance Publique des Hôpitaux de Paris)'
+date: 2022-09-05
+tags: user-story
+synopsis: 'Quarkus adoption by APHP (Assistance Publique des Hôpitaux de Paris).'
+author: jean-Yves Terrien
```

Comment:
The author should be added here: https://github.com/quarkusio/quarkusio.github.io/blob/develop/_data/authors.yaml

And then you can use the username to reference it in the `author` attribute.